### PR TITLE
Added testimonials feature

### PR DESCRIPTION
**Pull request by Zienab120**

### CODE REVIEW

File: app/Models/Product.php
Comment (Huma-volve) [inline app/Models/Product.php:1]: 1-product model do not have testimonials
2- remove this relation please

File: app/Models/Testimonials.php
Comment (Huma-volve) [inline app/Models/Testimonials.php:1]: also here not exist relation between product and Testimonials

File: database/migrations/2024_12_15_164403_create_testimonials_table.php
Comment (Huma-volve) [inline database/migrations/2024_12_15_164403_create_testimonials_table.php:1]: using migration to remove caption_text to text and media_type

File: app/Http/Controllers/TestimonialController.php
Comment (Huma-volve) [inline app/Http/Controllers/TestimonialController.php:1]: 1- first the testimonials is not belongs to products is belongs to our website meaning the customer saying review about the website
2- where is the api to get testimonials the front end will use it

**Review by ghost — CHANGES_REQUESTED** [review]
Follow review

**Pull request merged**